climpred/metrics.py

```diff
@@ -1749,25 +1713,26 @@ def _brier_score(forecast, verif, dim=None, **metric_kwargs):
         +-----------------+-----------+
 
     Reference:
-        * Brier, Glenn W. Verification of forecasts expressed in terms of
-          probability.” Monthly Weather Review 78, no. 1 (1950).
-          https://doi.org/10.1175/1520-0493(1950)078<0001:VOFEIT>2.0.CO;2.
         * https://www.nws.noaa.gov/oh/rfcdev/docs/
           Glossary_Forecast_Verification_Metrics.pdf
+        * https://en.wikipedia.org/wiki/Brier_score
 
     See also:
         * properscoring.brier_score
         * xskillscore.brier_score
 
     Example:
-        >>> def pos(x): return x > 0
-        >>> compute_perfect_model(ds, control, metric='brier_score', logical=pos)
+        >>> def pos(x): return x > 0  # checking binary outcomes
+        >>> hindcast.verify(metric='brier_score', comparison='m2o', \
+                dim='member', alignment='same_verifs', logical=pos)
+        >>> hindcast.map(pos).mean('member').verify(metric='brier_score', \
+                comparison='e2o', dim=[], alignment='same_verifs')
```

Comment:
Great! Maybe add a comment that these are two different approaches.  Or you can split them into two sections:


```python
Example:
    Define a boolean/logical function for binary scoring:
        
    >>> def pos(x): return x > 0

    Option 1. Pass with keyword `logical`:

    >>> hindcast.verify(metric='brier_score',
                                     comparison='m2o',
                                     dim='member',
                                     alignment='same_verifs',
                                     logical=pos)

    Option 2. Pre-process to generate a binary forecast and verification product.

    >>> hindcast.map(pos).verify(metric='brier_score', comparison='e2o', dim=[], alignment='same_verifs')
```

Reply:
Some clarifying points:

1. Are you showing between the two options that you can do this over the member dimension (in the first) and ensemble mean dimension (in the second)? Maybe clarify that in the Option 1, Option 2 text.
2. Is there any reason you are showing `.mean('member')` in option 2? I think it adds unnecessary complexity, since `climpred` will handle that for them.
3. Why is `dim=[]` in the second option?

Reply:
thanks for those comments. added Option 3.